contxt/auth/token.py

```diff
@@ -25,14 +23,10 @@ class AuthTokenValidator(NamedTuple):
 
     def validate(self, token: str) -> TokenPayload:
         try:
-            return jwt.decode(token=token, key=self.public_key, algorithms=ALGORITHMS.RS256, audience=self.audience,
+            return jwt.decode(jwt=token, key=self.public_key, algorithms="RS256", audience=self.audience,
                               issuer=self.issuer)
-        except jwt.ExpiredSignatureError:
-            raise AuthError("TOKEN_EXPIRED: token is expired")
-        except jwt.JWTClaimsError:
-            raise AuthError("INVALID_CLAIMS: please check the audience and issuer")
-        except jwt.JWTError:
-            raise AuthError("INVALID_TOKEN: please check the token")
+        except jwt.PyJWTError as e:
```

Comment:
Ideally we'd expect this error to be as descriptive as possible.

Reply:
This is just the base class for all `pyjwt` errors. The actual errors raised are indeed descriptive.